File: server/models/rocket.js
"use strict";

class Rocket {

    constructor (id, name, defaultPads, family, wikiURL, infoURLs, imageURL, imageSizes) {
        this.id = id;
        this.name = name;
        this.defaultPads = defaultPads;
        this.family = family;
        this.wikiURL = wikiURL;
        this.infoURLs = infoURLs;
        this.imageURL = imageURL;
        this.imageSizes = imageSizes;
    }

    static fromJSON (json) {
        let x = JSON.parse(json);
        return new Rocket(x.id, x.name, x.defaultPads, x.family, x.wikiURL, x.infoURLs, x.imageURL, x.imageSizes)
    }
}

File: src/main/java/com/eccff/launchbot/models/Rocket.java
package com.eccff.launchbot.models;

import lombok.Builder;
import lombok.Data;

import javax.persistence.ElementCollection;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.ManyToOne;
import java.util.List;

/**
 * https://launchlibrary.net/1.2/rocket
 */

@Data
@Entity
public class Rocket {

    @Id
    private int id;
    private String name;
    private String defaultPads;
    @ManyToOne(optional = false)
    private RocketFamily family;
    private String wikiURL;
    @ElementCollection(targetClass=String.class)
    private List<String> infoURLs;
    private String imageURL;
    @ElementCollection(targetClass=Integer.class)
    private List<Integer> imageSizes;

}